load-agent/locustClient.py

```diff
@@ -261,7 +286,11 @@ def accept_invite(self, invite, useConnectionDid=False):
             self.run_command({"cmd": "receiveInvitation", "invitationUrl": invite})
 
         line = self.readjsonline()
-
+    
+        # Filter out unwanted lines
+        if line.get("connection") is None:
```

Comment:
I don't completely understand what is going on here but the new protocol added output here unrelated to the connection. Ignoring unwanted output fixed the problem but all the locust files also needed to ignore the invalid return value.